a2x.py

```diff
@@ -364,11 +364,21 @@ def parse_options():
     result = []
     if os.path.isfile(asciidoc_file):
         options = ''
-        with open(asciidoc_file) as f:
+        with open(asciidoc_file, 'rb') as f:
+            line_number = 0
             for line in f:
-                mo = re.search(r'^//\s*a2x:', line)
+                line_number += 1
+                mo = re.search(b'^//\s*a2x:', line)
                 if mo:
-                    options += ' ' + line[mo.end():].strip()
+                    try:
+                        options += ' ' + line[mo.end():].strip().decode('ascii')
+                    except UnicodeDecodeError as e:
+                        #Encountered a problem decoding an option for a2x.
+                        #Discard this option and continue on with a warning.
+                        warning("Options must be in %s " % e.encoding  +
+                                "encoding: File '%s' " % asciidoc_file +
+                                "line number '%s' " %line_number       +
+                                "error '%s'." % e.reason)
```

Comment:
I would probably not bother outputting the exception reason as a non-programmer using asciidoc is not going to know what it means really. I would make it a bit simpler:
```suggestion
                        warning(
                            "Could not decode option to %s " % e.encoding +
                            "on line %s in %s" % (line_number, asciidoc_file)
                        )
```